manufacturing/enclosed_build/README.md

```diff
@@ -17,7 +17,34 @@ Here is the CAD model for viewing: [autodesk360 link](https://a360.co/2PTyRzh)
 
 This page contains a running list of notes and planned updates that will be organized and added to as we get further into beta design and build.
 
-### Venturi Updates
+The purpose of the Beta Build is to build the complete ventilator in an enclosure.
+
+The Components/Sub assemblies of the Beta Build is as follows:
+
+Top Assy - This acts as the highest assembly, it includes all of the parts and sub assemblies to complete the beta build. (includes image of assy, link to CAD, links to stp files, etc) #TODO
+
+Blower Sub Assembly
```

Comment:
Should this be a bullet list?
Are these meant to be links? If so, can they already link to the pizza-oriented sub-assemblies or are we meant to create separate pages for this?
Just, for example, if I stumble onto this page and I want to improve it, I am not sure I get an idea of what is expected.
Can we assume the guidelines [here](https://github.com/RespiraWorks/Ventilator/tree/master/manufacturing#sub-assemblies) apply?